services/py-api/src/service/hackathon_service.py

```diff
@@ -65,6 +65,24 @@ async def create_random_participant(
         # As when first created, the random participant is not assigned to a team we return the team as None
         return Ok((result.ok_value, None))
 
+    async def verify_admin_participant_and_team_in_transaction(self, admin_id: str) -> Any:
+        return await self._tx_manager.with_transaction(self.verify_admin_participant_and_team_callback, admin_id)
+
+    async def verify_admin_participant_and_team_callback(
+        self, admin_id: str, session: Optional[AsyncIOMotorClientSession] = None
+    ) -> Any:
+        result_admin = await self._participant_repo.update(
+            obj_id=admin_id, input_data={"email_verified": True}, session=session
```

Comment:
<img width="902" alt="image" src="https://github.com/user-attachments/assets/edfd9297-1d85-4828-b89b-71ca193c2da6" />
Fix the type, please.